esprima.js

```diff
@@ -11,6 +11,11 @@
 var esprima = require('esprima-fb');
 var estraverse = require('estraverse-fb');
 var escodegen = require('escodegen');
+var listver = require('listver');
+
+var webpackMajorVersion = parseInt(
```

Comment:
@larkin so just to double check, I can rely on loaderCtx.Version instead of  webpackMajorVersion for this check?